content/docs/1.9.0/important-notes/_index.md

```diff
@@ -58,6 +60,25 @@ The `v1beta1` version of the Longhorn API is deprecated in v1.9.0 and will be re
 
 Deprecated APIs are no longer served and may therefore cause unexpected or unwanted behavior. Avoid using longhorn.io/v1beta1 in new code and, if possible, rewrite existing code to exclude this version.
 
+## Breaking Change
+
+### V2 Backing Image
+
+Due to naming conflicts in the extended attributes (xattrs) used by SPDK backing image logical volumes, the V2 backing image in version 1.9.0 and later is incompatible with earlier versions.
+
+- **Before upgrading to v1.9.0**:
+  - Ensure the backup targets are working.
+  - Create full backup for all volumes created using a V2 backing image.
+  - Detach and delete these volumes after backup.
+  - In backing image page, save all the spec of V2 backing images, including the name and all parameters.
+  - Delete all V2 backing image.
```

Comment:
```suggestion
  - Create backups for volumes using V2 backing images.
  - Detach and delete these volumes after backup.
  - Delete V2 backing images.
```

Reply:
Maybe we should emphasize that the backing image are going to be recreated using the same parameters here?